Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -3427,6 +3452,7 @@ def gen2022HiMix(fragment,howMuch):
 
 steps['HARVESTDR3']  = merge([{'--conditions':'auto:run3_data','--era':'Run3'}, steps['HARVESTD']])
 steps['HARVESTDR3_reHLT']  = merge([{'--conditions':'auto:run3_data_relval','--hltProcess':'reHLT','-s':'HARVESTING:@standardDQM+@miniAODDQM'}, steps['HARVESTDR3']])
```

Comment:
See comment above about `FakeHLT` DQM/harvesting sequences.

Reply:
Fwiw, I would find it clearer to have a clear distinction betwen "2022" (e.g. `HARVESTDR3_reHLT_2022`) and "2023"  (e.g. `HARVESTDR3_reHLT_2023`), instead of a generic "Run3". The 2022 workflows would use `Run3` for now, but they would eventually use `Run3_2022` if that is introduced (which I thought was the plan, see https://github.com/cms-sw/cmssw/pull/41271#issuecomment-1506968258).

Reply:
Yes, this is how I started propagating the changes but eventually decided to stick to the generic Run3 :) since we do not have the Run3_2022 era. We can explicitly call the relevant steps 2022 and 2023, as you suggest.